lib/Extractor/KLEEBuilder.cpp

```diff
@@ -18,6 +18,7 @@
 #include "souper/Extractor/ExprBuilder.h"
 #include "llvm/ADT/SmallVector.h"
 #include "llvm/Analysis/LoopInfo.h"
+#include "llvm/Support/CommandLine.h"
```

Comment:
what's this for?

Reply:
The file uses cl:: namespace without including this header file. It worked with LLVM 9 probably because CommandLine.h was indirectly added by some other header file we are including here but this doesn't work with LLVM 10. Better to be explicit, as they say.